### PR TITLE
Split emysql.hrl

### PR DESCRIPTION
**Pull request by saa**
Maybe later, have more partitions.

### CODE REVIEW

File: include/emysql.hrl [hunk 1]
```diff
@@ -25,69 +25,32 @@
...
--define(CLIENT_MULTI_RESULTS, 131072).
--define(TRANSACTIONS, 8192).
--define(SECURE_CONNECTION, 32768).
--define(CONNECT_WITH_DB, 8).
--define(CONN_TEST_PERIOD, 28000).
 -define(TCP_RECV_BUFFER, 8192).
+-define(CONNECT_WITH_DB, 8).
 
-
-%% MYSQL TYPES
```
Comment (jlouis) [inline include/emysql.hrl:55]: We can't lift these out of emysql.hrl. They are _definitely_ part of what you need in order to transform field types in a `#result_packet{}`. I have at least one code-base I maintain (closed source) which use these explicitly, so moving them away is going to break stuff.

We'll probably need a note for doing this around 1.0.0, but not before. And even then, we should probably only break part of the API for each major version to give people some time to adapt.

**Comment by jlouis** [comment]
All the other changes look nice. We should get those in.

**Comment by jlouis** [comment]
Could you prepare a patch with only part of the changes, or should I do that?

**Comment by jlouis** [comment]
Bump :)

**Comment by saa** [comment]
Sorry, unfortunately stopped using mysql and no free time to devote to his project.

**Pull request closed without merging**